Levels/NewStarbase/NewStarbase.prefab

```diff
@@ -42290,12 +42337,37 @@
                 {
                     "op": "replace",
                     "path": "/Entities/Entity_[38393244868340]/Components/Component_[16396523672802690395]/configuration/propertyOverrides/overrides/0/Datum/value",
-                    "value": 14.0
+                    "value": 16.5
+                },
+                {
+                    "op": "replace",
+                    "path": "/Entities/Entity_[38393244868340]/Components/Component_[16396523672802690395]/configuration/propertyOverrides/overrides/1/Datum/value/MaterialAssetId/guid",
```

Comment:
This is a material asset override on a shield generator. Just want to make sure this was intentional..?